instrumentor/controllers/agentenabled/podswebhook/mount.go

```diff
@@ -8,10 +8,24 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+func checkIfMountDirectoryExists(containerSpec *corev1.Container, dir string) bool {
```

Comment:
I would just inline these functions with a comment about why you're doing this loop

Reply:
done